lib/utils.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const clone = require('rfdc')({ circles: true })
```

Comment:
Are there any other methods for resolving the issue at hand? This replacement module has not been updated in quite some time, has a long open dependabot PR, and https://github.com/planttheidea/fast-copy/issues/39#issuecomment-766400371

Reply:
I checked every library with deep cloning and a minimal user base but I couldn't find any other library that can clone errors and would pass every test

Reply:
Is there any other way, within this code base, outside of a cloning module?

Reply:
@davidmarkclements ping?

Reply:
I think we can ship this

Reply:
If I get a fix on rdfc, I'd be very happy to.

Reply:
which fix are you talking about?

Reply:
add support to rdfc to clone errors.